### ISSUE TITLE
SAGA processing algs not working 

### ISSUE THREAD

**Issue by PeterPetrik** [qgis/QGIS-Mac-Packager#76]

**Comment by PeterPetrik** [comment]
probably something like this is needed https://github.com/qgis/QGIS-Mac-Packager/blob/5f09aa9ff1fe63f59f92bd48615d38309cacf27e/qmp/bundler/fix_linker_paths.py#L225

**Comment by PeterPetrik** [comment]
looks like the problem is in hardcoded path to toolchains in 
```
./src/saga_core/saga_cmd/saga_cmd.cpp:407:		Load_Libraries(SG_File_Make_Path(CSG_String(SHARE_PATH), SG_T("toolchains")));	// look for tool chains
```

**Comment by PeterPetrik** [comment]
fixed in qgis-deps-0.5.4
